server/handler/pull_request.go

```diff
@@ -66,6 +67,21 @@ func (h *PullRequest) Handle(ctx context.Context, eventType, deliveryID string,
 		logger.Error().Err(errors.WithStack(err)).Msg("Error processing pull request")
 	}
 
+	if event.GetAction() == "labeled" || event.GetAction() == "opened" {
+		logger.Debug().Msgf("received pull_request %s event", event.GetAction())
+
+		base, _ := pullCtx.Branches()
+		ref, _, err := client.Git.GetRef(ctx, pullCtx.Owner(), pullCtx.Repo(), fmt.Sprintf("refs/heads/%s", base))
+		if err != nil {
+			return errors.Wrap(err, "could not get git reference of PR base branch")
+		}
+
+		baseRef := ref.GetObject().GetSHA()
+		if err := h.UpdatePullRequest(logger.WithContext(ctx), pullCtx, client, pr, baseRef); err != nil {
```

Comment:
The `baseRef` passed to `UpdatePullRequest` is a ref name rather than a SHA. You should be able to use the `fmt.Sprintf("refs/heads/%s", base)` directly, without resolving it.

Reply:
Nice, that will really simplify things